pkg/auth/autorest_auth.go

```diff
@@ -23,10 +23,23 @@ import (
 	"github.com/Azure/go-autorest/autorest"
 	"github.com/Azure/go-autorest/autorest/adal"
 	"github.com/Azure/go-autorest/autorest/azure"
-	klog "k8s.io/klog/v2"
+	"k8s.io/klog/v2"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/jongio/azidext/go/azidext"
 )
 
 func NewAuthorizer(config *Config, env *azure.Environment) (autorest.Authorizer, error) {
+	// TODO (charliedmcb): need to get track 2 support for the skewer API, and align all auth under workload identity in the same way within cred.go
+	if config.UseNewCredWorkflow {
+		klog.V(2).Infoln("auth: using workload identity for new authorizer")
```

Comment:
since we are touching the config code do you think we could remove instances of klog all together? Is there a reason we use klog rather than standard knativing contxt logging?

Reply:
I was just modeling off the surrounding code here. I'm not sure why klog was used around here before. I do think I'd save any change to the logging for a separate PR though for keeping focus/PRs small.

I did actually find that these log outputs were not being shown by default in standalone, which I'd personally consider a bug. I was going to surface fixing that after this PR.

Reply:
Ok we can solve in a different PR